test/helpers/handshake_srp.c

```diff
@@ -49,6 +49,8 @@ int configure_handshake_ctx_for_srp(SSL_CTX *server_ctx, SSL_CTX *server2_ctx,
         SSL_CTX_set_srp_username_callback(server_ctx, server_srp_cb);
         server_ctx_data->srp_user = OPENSSL_strdup(extra->server.srp_user);
         server_ctx_data->srp_password = OPENSSL_strdup(extra->server.srp_password);
+        if (server_ctx_data->srp_user == NULL || server_ctx_data->srp_password == NULL)
+            return 0;
```

Comment:
There will need to be some OPENSSL_free() calls here to avoid "lost" memory.

Reply:
Fine, I have sent the fixup commit.